src/pages/Confirmation.jsx

```diff
@@ -0,0 +1,149 @@
+import React, {useEffect, useState} from 'react';
+import Rating from 'react-rating';
+import styled from "styled-components";
+import Button from "../components/Button";
+import Card from "../components/QuestionCard";
+import axios from "../axios-instance";
+import StarOn from "assets/StarOnIcon.svg"
+import StarOff from "assets/StarOffIcon.svg"
+import {v4 as uuidv4} from "uuid";
+import { motion } from "framer-motion"
+
+const Header = styled.h2`
+  text-align: center;
+  font-weight: normal;
+  font-size: 30px;
+  padding: 44px 0px 67px 0px;
```

Comment:
```suggestion
  font-size: 36px;
  margin: 44px 0 67px 0;
```
Padding używa się raczej do wypełniania elementów np. puste miejsce wewnątrz buttonów miedzy borderem a tekstem. Do zwiększania miejsca miedzy elementami uzywa się zawsze marginu.
![PeSIJ](https://user-images.githubusercontent.com/39658211/81498884-c2a59480-92c7-11ea-9da2-cefa865409e8.gif)